yarn.lock

```diff
@@ -3,9 +3,9 @@
 
 
 "@babel/cli@^7.0.0":
-  version "7.4.3"
```

Comment:
You can make sure to rebase with the most current branch `develop` to verify that we are not modifying the `yarn.lock` unnecessarily.